libr/core/cmd_print.c

```diff
@@ -1629,6 +1629,7 @@ static void disasm_strings(RCore *core, const char *input, RAnalFunction *fcn) {
 	free (string2);
 	free (string);
 	free (s);
+	free (str);
```

Comment:
what about a variable to track when free it. Afterwards run some test with asan or valgrind such as t.esil/apple iirc that path is triggered.